src/frontend/src/utils/reimbursement-request.utils.ts

```diff
@@ -124,8 +124,10 @@ export const cleanReimbursementRequestStatus = (status: ReimbursementStatusType)
     case ReimbursementStatusType.DENIED: {
       return 'Denied';
     }
-    default:
+    case ReimbursementStatusType.PENDING_LEADERSHIP_APPROVAL:
```

Comment:
Why do these cases not have { } but the other ones do, it should be all either have them or none of them have them (I think it looks better if all of them don't have the brackets)